pyproject.toml

```diff
@@ -18,15 +18,14 @@ minversion = "6.0"
 addopts = "--verbose --doctest-modules -ra --cov package"  # Consider adding --pdb
 doctest_optionflags = "IGNORE_EXCEPTION_DETAIL"
 testpaths = [
-    "package",
     "tests",
 ]
 
 # https://github.com/pytest-dev/pytest-cov
 # https://github.com/nedbat/coveragepy
 [tool.coverage.run]
 omit = [
-    "package/__main__.py",
+    "*/package/__main__.py",
```

Comment:
Why not `src/package/__main__.py`?

Reply:
I think because if the `src/` folder’s content is packaged and installed for tox then it’s base path has changed.

Reply:
> > Now that there is a `src` dir, I wonder if it would be helpful to explicitly enable/disable flags for `src` in `mypy.ini`
> 
> Any suggestions you’d like to see here?

Perhaps the strict checks could be only enabled for `src`?
```
# Strict checks.
[mypy-src.*]
disallow_untyped_calls=True
disallow_untyped_defs=True
disallow_incomplete_defs = True
disallow_untyped_decorators = True
```

Reply:
Something like this?
```diff
diff --git a/mypy.ini b/mypy.ini
index 7027aa7..160d55f 100644
--- a/mypy.ini
+++ b/mypy.ini
@@ -7,14 +7,16 @@ platform=linux
 
 # Useful settings.
 show_column_numbers=True
-
-# Strict checks.
-disallow_untyped_calls=True
-disallow_untyped_defs=True
-disallow_incomplete_defs = True
-disallow_untyped_decorators = True
+check_untyped_defs=True
 
 # And some more warnings.
 warn_return_any=True
 warn_redundant_casts=True
 warn_unused_ignores=True
+
+# Per-module options: strict checks.
+[mypy-src.*]
+disallow_untyped_calls=True
+disallow_untyped_defs=True
+disallow_incomplete_defs = True
+disallow_untyped_decorators = True
```

Reply:
Yes, that way we have better control of the checks per module.

Reply:
Commit https://github.com/jenstroeger/python-package-template/pull/27/commits/72b0269dbd884fe4be8f372efe98f8fea8f5c94e.